server/pkg/utils/email/email.go

```diff
@@ -18,6 +19,69 @@ import (
 	"github.com/spf13/viper"
 )
 
+func sendViaSMTP(toEmails []string, fromName string, fromEmail string, subject string, htmlBody string, inlineImages []map[string]interface{}) error {
```

Comment:
nit: It's nice to push private methods to the bottom of the file, so that if you're using an IDE that shows an outline of the methods within the file, you can easily find out the public APIs it exposes